source/_integrations/shopping_list.markdown

```diff
@@ -59,18 +59,36 @@ Clear completed items from the shopping list.
 
 ## Using in Automations
 
-The simplest way to use the shopping list with automations (e.g., when entering a zone with shops) is to create a notification that can be clicked to open the shopping list.
+A `shopping_list_updated` event is triggered when items in the list are modified, with the following data payload attached to it. This can be used to trigger automations such as sending a push notification when someone adds an item to the shopping list, which when clicked will open the list.
```

Comment:
```suggestion
A `shopping_list_updated` event is triggered when items in the list are modified, with the following data payload attached to it. This can be used to trigger automations such as sending a push notification when someone adds an item to the shopping list, which when clicked, will open the list.
```